flink-connector/src/main/java/org/apache/gravitino/flink/connector/PartitionConverter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.flink.connector;
+
+import java.util.List;
+import org.apache.gravitino.rel.expressions.transforms.Transform;
+
+/**
+ * The PartitionConverter is used to convert the partition between Flink and Gravitino. The Flink
+ * only support identity transform. Some of the table like Apache Paimon will use the table
+ * properties to store the partition transform, so we can implement this interface to achieve more
+ * partition transform.
+ */
+public abstract class PartitionConverter {
```

Comment:
interface?

Reply:
@FANNG1 I think the abstract class is OK. Other classes won't implement it. What do you think?

Reply:
interface is more proper here to me

Reply:
Done